api/scenario_api_autoware/include/scenario_api_autoware/scenario_api_autoware.h

```diff
@@ -72,15 +59,20 @@ struct Vehicle_Data
   double front_overhang;
   double rear_overhang;
   double vehicle_height;
-  double front_from_base() { return front_overhang + wheel_base; };
-  double rear_from_base() { return -rear_overhang; };
+  double max_longitudinal_offset;
+  double min_longitudinal_offset;
+  double max_height_offset;
+  double min_height_offset;
   double left_from_base() { return wheel_tread / 2.0 + wheel_width / 2.0; };
   double right_from_base() { return -(wheel_tread / 2.0 + wheel_width / 2.0); };
-  double top_from_base() { return vehicle_height; };
-  double bottom_from_base() { return 0.0; };
 };
 
-class ScenarioAPIAutoware
+namespace bg = boost::geometry;
+using Point = bg::model::d2::point_xy<double>;
+using Polygon = bg::model::polygon<Point>;
+using Line = bg::model::linestring<Point>;
+
+class ScenarioAPIAutoware : public rclcpp::Node
```

Comment:
It looks like ScenarioAPIAutoware is a library class used by `scenario_api` package. Making this inheriting rclcpp::Node class would make this a separate node. 

This should instead take `rclcpp::Node::SharedPtr` as an input to the constructor and use it for setting parameters and pub/subs.

Reply:
I see, thanks for the explanation! I will make the fix.

Reply:
See 56cdbc4